debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java

```diff
@@ -100,16 +100,17 @@ public Config validate(Map<String, String> connectorConfigs) {
                 && databaseValue.errorMessages().isEmpty()) {
             // Try to connect to the database ...
             try (PostgresConnection connection = new PostgresConnection(config.jdbcConfig())) {
-                try {
-                    connection.execute("SELECT version()");
-                    logger.info("Successfully tested connection for {} with user '{}'", connection.connectionString(),
-                            connection.username());
-                }
-                catch (SQLException e) {
-                    logger.info("Failed testing connection for {} with user '{}'", connection.connectionString(),
-                            connection.username());
-                    hostnameValue.addErrorMessage("Unable to connect: " + e.getMessage());
-                }
+                connection.execute("SELECT version()");
+                logger.info("Successfully tested connection for {} with user '{}'", connection.connectionString(),
+                    connection.username());
+            } catch (Throwable e) {
```

Comment:
Why is this changed to `Throwable`?

Reply:
The exception thrown is a RuntimeException wrapped in a ExecutionException. To be safe, I just made it catch throwable. Exact stack is this:
```
[ERROR] 2020-07-29 05:55:00,853 [qtp702325149-68] org.apache.kafka.connect.runtime.rest.errors.ConnectExceptionMapper toResponse
 - Uncaught exception in REST call to /connector-plugins/io.debezium.connector.postgresql.PostgresConnector/config/validate
java.lang.RuntimeException: Couldn't obtain encoding for database dev
        at io.debezium.connector.postgresql.connection.PostgresConnection.determineDatabaseCharset(PostgresConnection.java:434)
        at io.debezium.connector.postgresql.connection.PostgresConnection.<init>(PostgresConnection.java:77)
        at io.debezium.connector.postgresql.connection.PostgresConnection.<init>(PostgresConnection.java:87)
        at io.debezium.connector.postgresql.PostgresConnector.validate(PostgresConnector.java:102)
        at org.apache.kafka.connect.runtime.AbstractHerder.validateConnectorConfig(AbstractHerder.java:356)
        at org.apache.kafka.connect.runtime.rest.resources.ConnectorPluginsResource.validateConfigs(ConnectorPluginsResource.jav
a:82)
        at jdk.internal.reflect.GeneratedMethodAccessor107.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```

Reply:
I see, do you foresee any situation where catching the exception and swallowing is potentially not good?